include/onnxruntime/core/framework/kernel_def_builder.h

```diff
@@ -258,6 +263,14 @@ class KernelDefBuilder {
     return *this;
   }
 
+  /**
+     Specify that this kernel's outputs are passed from external.
+  */
```

Comment:
Not clear what 'from external' means (at least if you're coming from the inferencing world). Can you clarify the description here?

Reply:
addressed.